### DISCUSSION TITLE
how to add a version to an existing package

### DISCUSSION THREAD

**Discussion by tunguskar** [microsoft/winget-cli#3132] [Q&A] [answered]
Hi,

I miss a version for this package:

https://winget.run/pkg/Python/Python.3.9

Version 3.9.8 is missing: https://www.python.org/downloads/release/python-398/

How is it possible to add this version?

**Comment by Trenly** [comment, marked as answer]
> Hi,
> 
> I miss a version for this package:
> 
> [winget.run/pkg/Python/Python.3.9](https://winget.run/pkg/Python/Python.3.9)
> 
> Version 3.9.8 is missing: [python.org/downloads/release/python-398](https://www.python.org/downloads/release/python-398/)
> 
> How is it possible to add this version?

You can use a tool like [WingetCreate](/microsoft/winget-create) or [YamlCreate](/microsoft/winget-pkgs/blob/master/doc/tools/YamlCreate.md) to add the version yourself. More information is in the guidelines on [Submitting A Package](/microsoft/winget-pkgs#submitting-a-package). Otherwise, if you submit a request for a package update at [winget-pkgs](/microsoft/winget-pkgs/issues/new/choose) someone will likely add the package soon.

**Comment by tunguskar** [comment]
Thank you Trenly
